frontend_v2/src/app/components/challenge/challengesettings/challengesettings.component.html

```diff
@@ -96,9 +96,32 @@ <h5 class="fw-light">Challenge Settings</h5>
             [phaseSelected]="phaseSelected()"
             #phaseselect
           ></app-selectphase>
-         </div>
-      </div>
+
+        <div class="row margin-bottom-cancel phase-card" *ngIf="selectedPhase">
+          <div class="col s12">
+            <button
+              class="btn ev-btn-dark waves-effect waves-dark grad-btn grad-btn-dark fs-12"
+              (click)="editPhaseDetails()"
+            >
+            Phase Details
+            </button>
+            <button
+              class="btn ev-btn-dark waves-effect waves-dark grad-btn grad-btn-dark fs-12"
+              (click)="togglePhaseVisibility()"
+            >
+              <i class="{{ phaseVisibility.icon }}"></i> Phase Visibility
+            </button>
+            <button
+              class="btn ev-btn-dark waves-effect waves-dark grad-btn grad-btn-dark fs-12"
+              (click)="toggleSubmissionVisibility()"
+            >
+              <i class="{{ submissionVisibility.icon }}"></i> Submission Visibility
```

Comment:
Same here! Please bind it with the selected phases's submission visibility.

Reply:
I believe we don't need to bind it to the particular phase, it's just that I missed the `else` statement.